src/custom_components/slider.scss

```diff
@@ -1,8 +1,7 @@
-.virtual-slider {
+.v-slider {
```

Comment:
Why do we have a slider2 which we are including and a slider which we are not including?
Unless there's a good reason, please remove slider2 and add slider to the _all file

Reply:
Resolved